### PR TITLE
Add Muffin/Slug

### PR DESCRIPTION
**Pull request by ADmad**
/cc @jadb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,6 +309,7 @@ Focus is on the specific task.
 *Search Engine Optimization*
 
 - [Tools:Slugged](https://github.com/dereuromark/cakephp-tools) - Containing Slugged behavior to auto-generate URL-compatible slugs from titles.
+- [Slug plugin](https://github.com/UseMuffin/Slug) - A plugin to generating slug and finding records by slug. Uses a pluggable architecture which allows using your own slug generator class.
```
Comment (dereuromark) [inline README.md]: "to generate slugs"

Comment (ADmad) [inline README.md]: Doh grammar fail.

**Comment by ADmad** [comment]
Fixed

**Comment by jadb** [comment]
nice!

File: README.md [hunk 2]
```diff
@@ -309,6 +309,7 @@ Focus is on the specific task.
 *Search Engine Optimization*
 
 - [Tools:Slugged](https://github.com/dereuromark/cakephp-tools) - Containing Slugged behavior to auto-generate URL-compatible slugs from titles.
+- [Slug plugin](https://github.com/UseMuffin/Slug) - A plugin for generating slugs and finding records by slug. Uses a pluggable architecture which allows using your own slug generator class.
```
Comment (jadb) [inline README.md]: Alphabetical order :stuck_out_tongue_winking_eye:

**Pull request merged**